AmazonChimeSDK/AmazonChimeSDKTests/audiovideo/data/transcript/TranscriptResultTests.swift

```diff
@@ -30,5 +33,7 @@ class TranscriptResultTests: XCTestCase {
         XCTAssertEqual(transcriptResult.startTimeMs, startTimeMs)
         XCTAssertEqual(transcriptResult.endTimeMs, endTimeMs)
         XCTAssertEqual(transcriptResult.alternatives, [])
+        XCTAssertEqual(transcriptResult.languageCode, languageCode)
+        XCTAssertEqual(transcriptResult.languageIdentification, [])
     }
```

Comment:
Same comment

Reply:
Have test cases with combinations of params, including null value.